pkg/sources/adapter/azureservicebussource/adapter.go

```diff
@@ -104,13 +107,20 @@ func NewAdapter(ctx context.Context, envAcc pkgadapter.EnvConfigAccessor, ceClie
 		logger.Panicw("Unable to parse entity ID "+strconv.Quote(env.EntityResourceID), zap.Error(err))
 	}
 
-	ns, err := servicebus.NewNamespace(namespaceFromEnvironment(entityID))
+	entityPath := entityPath(entityID)
```

Comment:
This variable isn't used anywhere anymore, except in the error message at line 125. We could get rid of the variable, and just inline `strconv.Quote(entityPath(entityID))` inside the error message.